probe/endpoint/procspy/proc_linux.go

```diff
@@ -22,8 +22,19 @@ var (
 	procRoot               = "/proc"
 	namespaceKey           = []string{"procspy", "namespaces"}
 	netNamespacePathSuffix = ""
+	ipv6IsSupported        = tcp6FileExists()
 )
 
+func tcp6FileExists() bool {
+	filename := filepath.Join(procRoot, "self/net/tcp6")
+	f, err := fs.Open(filename)
+	if err != nil {
```

Comment:
Being a proc file (living in the kernel's memory), I don't think we gain much by adding this refinement.